AEPEdge.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "AEPEdge"
-  s.version          = "4.2.0"
+  s.version          = "4.3.0"
```

Comment:
make sure you also update the Package.swift one or simply run the update versions workflow